pkg/crypto/crypto.go

```diff
@@ -29,10 +29,11 @@ func EncryptVolume(devicePath, passphrase, keyCipher, keyHash, keySize, pbkdf st
 }
 
 // OpenVolume opens volume so that it can be used by the client.
-func OpenVolume(volume, devicePath, passphrase string) error {
-	devPath := types.GetVolumeDevicePath(volume, true)
-	if isOpen, _ := IsDeviceOpen(devPath); isOpen {
-		logrus.Debugf("Device %s is already opened at %s", devicePath, devPath)
+// attachingVolumeHostPath is the path of the volume on the host that will be opened.
```

Comment:
Why rename the parameter? The original one is much easier to understand.

Reply:
As @shuo-wu mentioned here https://github.com/longhorn/longhorn-manager/pull/3438#discussion_r1909857928.
It might be confusing between the devices we want to open and format.
I will revert it and leave the comment there.

Reply:
Reverted it. Thanks.